### ISSUE TITLE
Bug: Assert.That(groups[0].NullableBoolean, Is.True); does not convert

### ISSUE THREAD

**Issue by inyutin-maxim** [meziantou/Meziantou.FluentAssertionsAnalyzers#6]
I have assertion:
```csharp
Assert.That(groups[0].IsMember, Is.True);
```
Where the property `IsMember` is declared as nullable boolean 

can be convert to:
```csharp
groups[0].IsMember.Should().BeTrue();
```
but convertion does not work

**Comment by meziantou** [comment]
Can you provide a repro? What is the error message?

A simple repro works on my side:

````c#
// Nunit
var groups = new[] { new { IsMember = true } };
Assert.That(groups[0].IsMember, Is.True);
````


````c#
// FluentAssertions
var groups = new[] { new { IsMember = true } };
groups[0].IsMember.Should().BeTrue();
````

**Comment by inyutin-maxim** [comment]
> Can you provide a repro? What is the error message?
> 
> A simple repro works on my side:
> 
> ```cs
> // Nunit
> var groups = new[] { new { IsMember = true } };
> Assert.That(groups[0].IsMember, Is.True);
> ```
> 
> ```cs
> // FluentAssertions
> var groups = new[] { new { IsMember = true } };
> groups[0].IsMember.Should().BeTrue();
> ```

Hi @meziantou  Your example works correctly because you have a Boolean field defined as non nullable, in my example the IsMember property is defined as nullable

**Comment by meziantou** [comment]
Fixed by f4b6bc8fdca90e8fa57bf0f0f2801450e2760fb9
